bundles/org.openhab.binding.gree/src/main/resources/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,137 @@
+		<label>GREE Air Conditioner</label>
+		<description>This Thing represents a GREE Air Conditioner</description>
+
+		<channels>
+			<channel id="power" typeId="system.power" />
+			<channel id="mode" typeId="mode" />
+			<channel id="temperature" typeId="temperature" />
+			<channel id="air" typeId="air" />
+			<channel id="dry" typeId="dry" />
+			<channel id="turbo" typeId="turbo" />
+			<channel id="windspeed" typeId="windspeed" />
+			<channel id="swingvertical" typeId="swingvertical" />
+			<channel id="powersave" typeId="powersave" />
+			<channel id="light" typeId="light" />
+			<channel id="health" typeId="health" />
+		</channels>
+
+		<config-description>
+			<parameter name="ipAddress" type="text" required="true">
+				<label>Device IP Address</label>
+				<description>IP-Address of the GREE device.</description>
+				<context>network-address</context>
+			</parameter>
+			<parameter name="broadcastAddress" type="text" required="false">
+				<label>Subnet Broadcast Address</label>
+				<description>Broadcast IP address of the local subnet.</description>
+				<context>network-address</context>
+			</parameter>
+			<parameter name="refresh" type="integer" required="true" unit="s">
+				<label>Update Interval</label>
+				<description>Interval to query an update from the device.</description>
+				<default>60</default>
+				<unitLabel>seconds</unitLabel>
+				<advanced>true</advanced>
+			</parameter>
+		</config-description>
+
+	</thing-type>
+
+	<channel-type id="mode">
+		<item-type>String</item-type>
+		<label>Mode</label>
+		<description>Sets the operating mode of the Air Conditioner - one of auto/cool/dry/fan/heat/eco/turbo or 0/1/2/3/4 or on/off</description>
+		<state>
+			<options>
+				<option value="auto">Auto</option>
+				<option value="cool">Cool</option>
+				<option value="eco">Eco</option>
+				<option value="dry">Dry</option>
+				<option value="fan">Fan</option>
+				<option value="turbo">Turbo</option>
+				<option value="heat">Heat</option>
+				<option value="ont">ON</option>
+				<option value="off">OFF</option>
+				<option value="0">Auto</option>
+				<option value="1">Cool</option>
+				<option value="2">Dry</option>
+				<option value="3">Fan</option>
+				<option value="4">Heat</option>
+			</options>
+		</state>
+	</channel-type>
+	<channel-type id="turbo">
+		<item-type>Switch</item-type>
+		<label>Turbo</label>
+		<description>Set on/off the Air Conditioner's Turbo mode.</description>
+	</channel-type>
+	<channel-type id="air">
+		<item-type>Switch</item-type>
+		<label>Air</label>
+		<description>Set on/off the Air Conditioner's Air function if applicable to the Air Conditioner model.</description>
+	</channel-type>
+	<channel-type id="dry">
+		<item-type>Switch</item-type>
+		<label>Dry</label>
+		<description>Set on/off the Air Conditioner's Dry function if applicable to the Air Conditioner model .</description>
+	</channel-type>
+	<channel-type id="temperature">
+		<item-type>Number</item-type>
```

Comment:
You could use Units of Measure here and add a state tag: https://www.openhab.org/docs/developer/bindings/thing-xml.html#state-description
```suggestion
		<item-type>Number:Temperature</item-type>
```

Reply:
done